src/parse_file.py

```diff
@@ -204,6 +204,28 @@ def parse_ascii_directive(line: str, z: bool) -> bytes:
     return b"".join(ret)
 
 
+def parse_incbin(args: List[str], options: Options) -> Optional[bytes]:
+    if options.incbin_dir is None:
+        return None
+    try:
+        filename = args[0].strip("'\"")
+        offset = int(args[1].strip(), 0)
+        size = int(args[2].strip(), 0)
+    except ValueError:
+        raise DecompFailure(f"Could not parse asm_data .incbin directive: {args}")
+
+    try:
+        with (options.incbin_dir / filename).open("rb") as f:
+            f.seek(offset)
+            data = f.read(size)
+            if len(data) == size:
+                return data
+    except OSError:
+        pass
```

Comment:
would be nice to at least warn about the paths that have failed to be opened, for user awareness